tests/cli/workflow/fastq/test_fastq_base.py

```diff
@@ -25,30 +25,3 @@ def test_store_fastq_analysis(another_case_id: str, cli_runner, fastq_context, h
         )
         > 0
     )
-
-
-def test_store_available_fastq_analysis(
```

Comment:
you don't feel this test added some value?

Reply:
it was testing whether one cli command invoked the other. that test was not relevant anymore.